Tests/ClientTests/Frontend/Home/HistoryHighlights/HistoryHighlightsViewModelTests.swift

```diff
@@ -137,6 +137,100 @@ class HistoryHighlightsViewModelTests: XCTestCase {
         XCTAssertTrue(telemetry.recordedObjects.contains(.firefoxHomepage))
     }
 
+    func testDelete() {
+        setupSubject()
+        sut.delete(getItemWithTitle(title: "to-delete"))
```

Comment:
We sometime use `subject` or `sut`, should we decide for a standard or using both is fine?

Reply:
I prefer "subject" because acronyms are 🤷 , but using both is also okay with me.

Reply:
Using both doesn't really bother me as the meaning is clear. I default to subject because I also don't like acronyms. Saying that tho if anyone feels strongly about instituting a standard I'm happy to switch to whatever.